scripts/ci/update_server_names.py

```diff
@@ -0,0 +1,156 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import re
+import requests
+import sys
+import xml.etree.ElementTree as etree
+from translate.misc.xml_helpers import reindent
+
+MULLVAD_SERVER_LIST_API = "https://api.mullvad.net/public/relays/wireguard/v2"
+
+def fetch_server_list():
+    country_names = []
+    city_names = []
+
+    try:
+        response = requests.get(MULLVAD_SERVER_LIST_API)
+
+        if response.status_code == 200:
+            data = response.json()
+
+            locations = data.get('locations', None)
+            if locations:
+                for country_code, location in locations.items():
+                    country_code = country_code.split("-")[0]
+
+                    country_name = location.get("country", None)
+
+                    # We don't want duplicates
+                    # so let's check if this country code is in the list already
+                    found_country = [c for c in country_names if c.get("code") == country_code]
+                    if country_name and not found_country:
+                        country_names.append({
+                            "name": country_name,
+                            "code": country_code
+                        })
+
+                    city = location.get("city", None)
+                    if city and city not in city_names:
+                        city_names.append(city)
+
+        else:
+            response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print("Error fetch server list:", e)
+    except ValueError as e:
+        print("Error parsing server list:", e)
+
+    return {
+        "countries": country_names,
+        "cities": city_names,
+    }
+
+
+def pascalize(string):
+    output = ""
+    for chunk in string.split(" "):
+        output += chunk[0].upper()
+        output += chunk[1:]
+    return output
+
+
+def get_city_translation_id(str):
+    # Remove the state suffix e.g. São Paulo, SP -> São Paulo
+    str = str.split(",")[0]
```

Comment:
```suggestion
    str = str.split(",")[0].strip()
```

Probably good to avoid traling spaces.